features/other_a/preferences_edit.feature

```diff
@@ -50,10 +50,8 @@ Feature: Edit preferences
   Then I should not see "My email address"
     And I should not see "My birthday"
   When I am logged in as "editname" with password "password"
-  Then I should see "Hi, editname!"
-    And I should see "Log Out"
-  When I post the work "This has two chapters"
-  And I follow "Add Chapter"
+    And I post the work "This has two chapters"
```

Comment:
Oof, this scenario. Testing way too many preferences at once. 😭 

I think we can take "personal details" out of the scenario description since we've stopped testing the birthdate and email address parts in this scenario (thank you for that!).

And I think we can streamline and update the work/chapter parts using some of the new steps that have been defined in the time since these tests were original written:

* `And I post the 2 chapter work "This has two chapters"` instead of manually going through and adding the second chapter here
* `Then I should be on the 2nd chapter of the work "This has two chapters"` and `Then I should  be on the 1st chapter of the work "This has two chapters"` instead of checking the chapter title on lines 58 and 60 to determine we're seeing the proper chapter
* And I _think_ we can do `Then I should be on the work "This has two chapters" in full mode` for the final check on line 85

Reply:
We couldn't do that for the final line, because it was expecting the query param in the URL, hence now checking for the lack of a "Next Chapter" text. Yes, negatives can be deceitful if we ever change the next chapter text, but we'd have to break both preferences and the text for that to bite us, and I think (hope) preferences are tested elsewhere as well, or will be by the time we break them...